scripts/packages/VSCodeServer/src/trees.jl

```diff
@@ -143,17 +170,7 @@ function assign_undefs(xs)
     return xs′
 end
 
-function treerender(x::Array{T,N}) where {T,N}
-    treerender(LazyTree(string(typeof(x), " with $(pluralize(size(x), "element", "elements"))"), wsicon(x), length(x) == 0, function ()
-        if length(x) > MAX_PARTITION_LENGTH
-            partition_by_keys(x, sz=MAX_PARTITION_LENGTH)
-        else
-            collect([SubTree(repr(k), wsicon(v), v) for (k, v) in zip(keys(x), vec(assign_undefs(x)))])
-        end
-    end))
-end
-
-function treerender(err, bt)
+function treerender(err::Exception, bt)
```

Comment:
Needs the `::Exception` because otherwise it overwrites a method definition above.